definitions/ext-tinyproxy/definition.yml

```diff
@@ -0,0 +1,58 @@
+# -- Mandatory fields --
+domain: EXT
+type: TINYPROXY
+
+synthesis:
+  # -- Mandatory fields --
+  name: targetName
+  identifier: targetName
+
+  conditions:
+    - attribute: metricName
+      prefix: tinyproxy_
+
+  tags:
+    - instrumentation.name
+
+  dashboardTemplates: ./dashboard.json
```

Comment:
Checks are failing because _dashboardTemplates_ doesn't exist yet, this field will be an array (instead of a string) after [this PR](https://github.com/newrelic-experimental/entity-synthesis-definitions/pull/11) though.